kola/tests/coretest/core.go

```diff
@@ -322,26 +322,41 @@ func TestServicesActive() error {
 	return servicesActive([]string{
 		"multi-user.target",
 		"docker.socket",
-		"systemd-timesyncd.service",
 		"update-engine.service",
+	}, []string{
+		"systemd-timesyncd.service",
+		"chronyd.service",
+		"ntpd.service",
 	})
 }
 
 func TestServicesActiveCoreOS() error {
 	return servicesActive([]string{
 		"multi-user.target",
-	})
+	}, []string{})
 }
 
-func servicesActive(units []string) error {
+func servicesActive(allOf []string, anyOf []string) error {
 	//t.Parallel()
-	for _, unit := range units {
+	for _, unit := range allOf {
 		c := exec.Command("systemctl", "is-active", unit)
 		err := c.Run()
 		if err != nil {
-			return fmt.Errorf("Services Active: %v", err)
+			return fmt.Errorf("services Active: %s: %v", unit, err)
+		}
+	}
+	anyActive := false
+	for _, unit := range anyOf {
+		c := exec.Command("systemctl", "is-active", unit)
+		err := c.Run()
+		if err == nil {
+			anyActive = true
+			break
 		}
 	}
+	if !anyActive {
+		return fmt.Errorf("no NTP service active")
```

Comment:
```suggestion
		return fmt.Errorf("no NTP service active (error: %v)", err)
```
That also works with a nil error shown as `<nil>` but I think we wouldn't run into this